include/dds/core/xtypes/DynamicData.hpp

```diff
@@ -190,6 +190,28 @@ class WritableDynamicDataRef : public ReadableDynamicDataRef
         return *this;
     }
 
+    template<typename T, class = PrimitiveOrString<T>>
+    WritableDynamicDataRef& operator = (
+            const T& other)
+    {
+        value(other);
+        return *this;
```

Comment:
I do not see an usage of the functionality that returning the `WritableDynamicDataRef` could offer. If the `DynamicData` is a sequence, I understand this: `data.push(1).push(5).push(42)`. But being a primitive what could be the usage? Could you write me an example?

Reply:
```cpp
DynamicData a(primitive_type<uint32_t>());
DynamicData b(primitive_type<uint32_t>());
b = a = 5;
```

Reply:
Ok, you convince me :D